xlive/H2MOD/Variants/Infection/Infection.cpp

```diff
@@ -222,6 +225,41 @@ void Infection::setPlayerAsZombie(int playerIndex) {
 	call_give_player_weapon(playerIndex, e_weapons_datum_index::energy_blade, 1);
 }
 
+void Infection::onGameTick()
+{
+	if(Engine::get_game_life_cycle() == _life_cycle_in_game && NetworkSession::LocalPeerIsSessionHost())
+	{
+		PlayerIterator playerIt;
+		int human_count = 0;
+		int zombie_count = 0;
```

Comment:
Make these two variables bytes since player limit is 16

Reply:
No